slack.tex

```diff
@@ -1,7 +1,53 @@
 \subsection{Slack}
 
-Description of the types of communications that happen on Slack
+The data management group uses instant messaging tools to answer questions, 
+share code snippets and errors, and coordinate activities in real-time.  
+Before using instant messaging, we would communicate primarily via e-mail or 
+over the phone. While these do still have their place, being able to have 
+conversations in a team-wide tool has been very beneficial. Discussions happen 
+with more team members participating, instead of just who happened to be on a 
+phone call or an e-mail thread. Questions and concerns have been able to be 
+answered much more quickly.
 
-Collaboration involvement
+We have used two different instant messaging tools in the last four years. The
+first tool we used was HipChat. Besides the essential features one would
+expect from an instant messaging client (such as discussion channels and one
+to one direct messaging), we were also able to receive Jenkins and JIRA issue
+notifications, which we found very helpful.
+
+We used HipChat for two years and eventually started looking at other options 
+because we found the user experience was, at times, inconsistent and 
+frustrating. One of the most significant issues was how it managed unread
+messages. If the user left the application, HipChat did not maintain where the
+user was in the message chain. When the tool restarted, it required the user
+to scroll back through messages to pick up where they left off. Notifications
+were erratic, and sometimes would arrive late or not at all. Some users
+particularly liked features such as video chat, while other users found this
+feature unstable and resorted to using other video chat services such as Google 
+Hangouts. 
+
+After a brief testing period, we decided to transition to Slack, which we
+continue to use today. The move from HipChat to Slack was straightforward,
+although we did lose the discussion history from HipChat, and Slack initially
+didn't have video chat (which it does now). We also took the opportunity to
+normalize the users' chat names to match the names used on GitHub, which
```

Comment:
This didn't work. We now ask that if they are different people can associate the github ID with their slack account and Jenkins bot will translate.

Reply:
Ok, a couple of options here;  I can mention our intent and subsequent solution, or leave this out.  If I keep it in and explain, I have a question:  Why didn't this work?

Reply:
I think people who came late to Github didn't get the name they wanted and saw no reason why they should be forced to use that name on Slack when they were able to choose anything they wanted.